tools/super-tox.py

```diff
@@ -445,6 +462,8 @@ def lxd_instance(
 @click.option("--repo", default=".*")
 @click.option("--fresh-tox/--no-fresh-tox", default=False)
 @click.option("-e", default=None, type=click.STRING)
+@click.option("--verbose/--no-verbose", default=False)
+@click.option("--sample", default=0)
```

Comment:
I should really add help strings to all of these ... but just to check, this is to stop after doing this many charms, yes?

Reply:
done